docs/serverless/rules/rules-ui-create.mdx

```diff
@@ -49,6 +49,12 @@ To create or edit ((ml)) rules, you need an appropriate user role. Additionally,
 
     1. The anomaly score threshold above which alerts are created.
 
+1. <DocBadge template="technical preview" /> (Optional) Use **Suppress alerts by** to reduce the number of repeated or duplicate alerts created by the rule. Refer to <DocLink slug="/serverless/security/alert-suppression">Suppress detection alerts</DocLink> for more information.
+
+    <DocCallOut title="Note">
+    Because ((ml)) rules generate alerts from anomalies, alert documents don't contain source event fields and you can only use anomaly fields when configuring alert suppression.
```

Comment:
```suggestion
    Because ((ml)) rules generate alerts from anomalies, which don't contain source event fields, you can only use anomaly fields when configuring alert suppression.
```